src/desktop/apps/partner/index.coffee

```diff
@@ -5,15 +5,15 @@ app = module.exports = express()
 app.set 'views', __dirname + '/templates'
 app.set 'view engine', 'jade'
 
-app.get '/:id', routes.requirePartner, routes.overview
-app.get '/:id/overview', routes.requirePartner, routes.redirectToOverview
-app.get '/:id/contact', routes.requirePartner, routes.contact
-app.get '/:id/about', routes.requirePartner, routes.about
-app.get '/:id/collection', routes.requirePartner, routes.collection
-app.get '/:id/shop', routes.requirePartner, routes.shop
-app.get '/:id/works', routes.requirePartner, routes.works
-app.get '/:id/shows', routes.requirePartner, routes.shows
-app.get '/:id/artists', routes.requirePartner, routes.artists
-app.get '/:id/artist/:artistId', routes.requirePartner, routes.artist
-app.get '/:id/articles', routes.requirePartner, routes.articles
+# app.get '/:id', routes.requirePartner, routes.overview
```

Comment:
We should add a comment to explain why all of these are commented out

Reply:
What if I remove them completely? I mean I don't see the sense to comment them out, because it wouldn't impact the code.

Reply:
I think it's fine to remove! I'd still add a little comment (maybe a `# TODO`?) to explain why we just have the articles path left. 

Something like `# TODO: Remove once we've rebuilt the partner article page in the new framework`

Reply:
Done